ci/steps/upload_test_results.py

```diff
@@ -0,0 +1,136 @@
+import dataclasses
+import json
+import os
+import sys
+from botocore.utils import merge_dicts
+
+import dacite
+import glci.model
+import glci.util
+import glci.s3
+
+
+def do_find(
```

Comment:
Is `do_find` a good name here? Maybe `find_release`.
Can't find a call to this function right now, though, so I'm not sure. Feel free to ignore this.